staff/lluis-bagur/app-template/about.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About</title>
+</head>
+<body>
+
+<header>
+<h1>About</h1>
+<nav>
+    <ul>
+        <li> <a href="register.html">Register</a></li>
+        <li> <a href="login.html">Login</a></li>
+        <li> <a href="home.html">Home</a></li>
+        <li> <a href="news.html">News</a></li>
+        <li> <a href="about.html">About</a></li>
+        <li> <a href="contact.html">Contact</a></li>
+        <li> <a href="profile.html">Profile</a></li>
+    </ul>
+</nav>
+
+</header>
+
+<main>
+    <article>
+        <header>
+                <h2>How we are?</h2>
+        </header>
+        <section>
+                <p> SKYLAB is an initiative of a network of professionals with more than 20 years of experience in the online sector.<br />
+                 Our Dean, David Monreal has worked for 12 years in InfoJobs with business and product development responsibilities.<br />
+                 He has the support of a Board of Counselors made up of professionals with over 25 years in the industry to ensure that<br />
+                 our training programs are tailored to the needs of the market.</p>
+
+        </section>
+
+        </article>
+    
+    
+    <article>
+        <header>
+            <h2>Community</h2>
+        </header>
+        <section>
+                <p> The vibrant tech scene that emerged attracts hundreds of startup companies to the city. There are plenty of <br />
+                    opportunities to carve out your place by launching or joining local businesses like InfoJobs, King, Skyscanner, <br />
+                    Atrápalo, Scytl, Privalia, Trovit, Marfeel, eDreams, amongst others. In our community you will also find investors <br />
+                    interested on promissing projects.</p>
+
+        </section>
+        
+        </article>
+
+
+</main>
+<footer>
```

Comment:
Empty section. Is this intentional?

Reply:
Yes, I did not know what to put, but I also created the structure, for future modifications.

Reply:
better add it in future modifications than empty